src/main/java/com/github/pires/hazelcast/HazelcastDiscoveryController.java

```diff
@@ -43,22 +55,24 @@
 
   @JsonIgnoreProperties(ignoreUnknown = true)
   static class Address {
-
     public String IP;
   }
 
   @JsonIgnoreProperties(ignoreUnknown = true)
   static class Subset {
-
     public List<Address> addresses;
   }
 
   @JsonIgnoreProperties(ignoreUnknown = true)
   static class Endpoints {
-
     public List<Subset> subsets;
   }
 
+  private static String getServiceAccountToken() throws IOException {
+    String file = "/var/run/secrets/kubernetes.io/serviceaccount/token";
```

Comment:
I don't think it's in 0.17.1

Brendan
On May 21, 2015 1:48 PM, "Paulo Pires" <notifications@github.com> wrote:

> In
> src/main/java/com/github/pires/hazelcast/HazelcastDiscoveryController.java
> <https://github.com/pires/hazelcast-kubernetes-bootstrapper/pull/2#discussion_r30827448>
> :
>
> >      public List<Subset> subsets;
> >    }
> >
> > +  private static String getServiceAccountToken() throws IOException {
> > +    String file = "/var/run/secrets/kubernetes.io/serviceaccount/token";
>
> It wasn't. 0.17.1 has just been tagged. Let me give it a try.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pires/hazelcast-kubernetes-bootstrapper/pull/2/files#r30827448>
> .
>